docs/api-docs/provider-apis/data-solutions/data-solutions-overview.md

```diff
@@ -0,0 +1,539 @@
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+
+{
+    "data": {
+        "id": 7,
+        "channel_id": 0,
+        "name": "Affiliate Conversion Tracking",
+        "enabled": false,
+        "connection": "<script>js code here...</script>"
+    },
+    "meta": {}
+}
+
+```
+
+```JSON title="Sample Response" lineNumbers
+{
+    "data": {
+        "id": 7,
+        "channel_id": 0,
+        "name": "Affiliate Conversion Tracking",
+        "enabled": false,
+        "connection": "<script>js code here...</script>"
+    },
+    "meta": {}
+}
+```
+
+&nbsp;
+
+## Get All Web Analytics
+
+All six web analytics will be returned. As shown, fields for codes (e.g. `verification_tag`) in which a merchant has not entered a value will return as an empty string.     
+
+```JSON title="Sample Request" lineNumbers
+GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/settings/analytics
+X-Auth-Token: {{ACCESS_TOKEN}}
+Content-Type: application/json
+Accept: application/json
+```
+
+```JSON title="Sample Response" lineNumbers
+{
+    "data": [
+        {
+            "id": 1,
+            "channel_id": 0,
+            "name": "Google Analytics",
+            "enabled": true,
+            "version": 2,
+            "data_tag_enabled": true,
+            "property_id": "GA-1234567890"
+        },
+        {
+            "id": 2,
+            "channel_id": 0,
+            "name": "Visual Website Optimizer",
+            "enabled": false,
+            "vwo_smartcode": "VWO-1234567890"
+        },
+        {
+            "id": 3,
+            "channel_id": 0,
+            "name": "Facebook Pixel",
+            "enabled": true,
+            "pixel_id": "FP-1234567890",
+            "is_oauth_connected": true
+        },
+        {
+            "id": 4,
+            "channel_id": 0,
+            "name": "Segment.com",
+            "enabled": false,
+            "api_key": "SEG-1234567890",
+            "data_tag_enabled": true
+        },
+        {
+            "id": 6,
+            "channel_id": 0,
+            "name": "Site Verification Tags",
+            "enabled": false,
+            "verification_tag": ""
+        },
+        {
+            "id": 7,
+            "channel_id": 0,
+            "name": "Affiliate Conversion Tracking",
+            "enabled": false,
+            "connection": "<script>js code here...</script>"
+        }
+    ],
+    "meta": {}
+}
+```
+
+&nbsp;
+
+## Related Resources
+- [Overview Web Analytics](https://support.bigcommerce.com/s/article/Data-Solutions?language=en_US#web-analytics)
+- [Google Analytics](https://support.bigcommerce.com/s/article/Setting-Up-Google-Analytics?language=en_US)
+- [Visual Website Optimizer](...)
+- [Facebook Pixel](https://support.bigcommerce.com/s/article/Facebook-Pixel?language=en_US)
+- [Segment](https://support.bigcommerce.com/s/article/Setting-up-Segment-com?language=en_US)
+- [Site Verification Tags](...)
```

Comment:
I assume `(...)` is a placeholder, but a reminder to add a URL

Reply:
It turns out we don't have a KB doc, so I'll remove the placeholder